packages/playwright/config.ts

```diff
@@ -0,0 +1,25 @@
+const isDevMode = process.env['isDev']
+type platform =
```

Comment:
I prefer using PascalCase for type names

```suggestion
type Project =
// or type Site =
```